### ISSUE TITLE
Generators do not run in Rails 3.2

### ISSUE THREAD

**Issue by justin808** [shakacode/react_on_rails#164]
@flynfish reported:

I'm running 3.2 and trying to run the generators it fails because of the `hide!` method:

```
[WARNING] Could not load generator "generators/react_on_rails/base_generator". Error: undefined method `hide!' for ReactOnRails::Generators::BaseGenerator:Class
```

A work around would be to copy/paste from these examples: https://github.com/shakacode/react_on_rails-generator-results

The runtime support for 3.2 is more important than the generator.

**Comment by flynfish** [comment]
PR #165 submitted

**Comment by flynfish** [comment]
I closed out PR #165 and submitted #182 as a better solution
